design/prototype-version-7/task-list/README.md

```diff
@@ -0,0 +1,140 @@
+# Create a form (task list page) page
+
+## Context
+
+When we tested version 6 of the prototype we introduced the task list page.  
+We tested the new page with form creators who had form building experience and some with low digital confidence. 
+
+We wanted to check users understanding of the process to get a form ‘live’ and how they navigate through the tasks. 
+
+This provided us with some valuable insights and formed the basis of version 7. 
+
+## What we tested last time
+
+![“Create a form” task list page. Screenshot](../screenshots/003-Create-a-form-Apply-for-a-juggling-licence.png)
```

Comment:
```suggestion
![“Create a form” task list page. Screenshot](../../prototype-version-6/screenshots/003-Create-a-form-Apply-for-a-juggling-licence.png)
```
I think this will link to V6 screenshot - if that’s correct

Reply:
yes it's meant to show the task list from V6 (which was tested)